packages/cli/src/commands/data-migrate.js

```diff
@@ -1,5 +1,5 @@
-export const command = 'dataMigrate <command>'
-export const aliases = ['dm']
+export const command = 'data-migrate <command>'
+export const aliases = ['dm', 'datamigrate']
```

Comment:
```suggestion
export const aliases = ['dm']
```

camelCase is automatically created by Yargs, which we need for backwards compatibility🤞